src/signals/incident-management/containers/IncidentDetail/components/LocationPreview/index.test.js

```diff
@@ -1,10 +1,62 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
 
-// import LocationPreview from 'index';
+import LocationPreview from './index';
+
+jest.mock('../MapDetail', () => () => <div data-testid="location-preview-map" />);
 
 describe('<LocationPreview />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+  let props;
```

Comment:
Doesn't need to be a `let`; value doesn't change.

Reply:
gedaan